### PR TITLE
Issue #1417: User URL alias not applied to admin user out of the box.

### LINKED ISSUE #1417: User URL alias not applied to admin user out of the box.
Although the URL path alias pattern for users exists (set to `accounts/[user:name]` out of the box), the admin user does not have an alias created (`user/1`).

The alias is created successfully after a bulk generate (`accounts/admin`), but should be there out of the box.

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/1417

### CODE REVIEW

**Comment by klonos** [comment]
Sandbox refresh...

**Comment by ghost** [comment]
@klonos Care to update this PR?

File: core/modules/user/tests/user_views_handler_field_name.test [hunk 1]
```diff
@@ -21,17 +21,20 @@ class UserViewsHandlerFieldNameTest extends ViewsSqlTest {
     $view->row_index = 0;
 
     $view->field['name']->options['link_to_user'] = TRUE;
+    $user = user_load(1);
+    $user_id = $user->uid;
+    $user_path = $user->path['alias'] ? $user->path['alias'] : 'user/' . $user->uid;
```
Comment (ghost) [inline core/modules/user/tests/user_views_handler_field_name.test]:
```suggestion
    $user_id = 1;
    $user = user_load($user_id);
    $user_path = $user->path['alias'] ? $user->path['alias'] : 'user/' . $user_id;
```

**Comment by klonos** [comment]
Done @BWPanda 👍🏼

File: core/modules/user/user.install [hunk 2]
```diff
@@ -1281,6 +1281,22 @@ function user_update_1020() {
...
+  if (module_exists('path')) {
+    $user_path_pattern = config_get('path.settings', 'user_pattern');
+    if ($user_path_pattern) {
+      $admin_user = user_load(1);
+      if ($admin_user->path['auto'] == FALSE) {
+        $admin_user->path['auto'] = TRUE;
+        user_save($admin_user);
+      }
+    }
+  }
```
Comment (ghost) [inline core/modules/user/user.install]:
```suggestion
  if (module_exists('path') && config_get('path.settings', 'user_pattern')) {
    $admin_user = user_load(1);
    if ($admin_user->path['auto'] == FALSE) {
      $admin_user->path['auto'] = TRUE;
      user_save($admin_user);
    }
  }
```

**Review by ghost — CHANGES_REQUESTED** [review]
Few suggested changes for simplification.

File: core/profiles/standard/standard.install [hunk 3]
```diff
@@ -407,10 +407,12 @@ function standard_install() {
   // Set this as the administrator role.
   config_set('system.core', 'user_admin_role', $admin_role->name);
 
-  // Assign user 1 the "administrator" role.
-  db_insert('users_roles')
-    ->fields(array('uid' => 1, 'role' => $admin_role->name))
-    ->execute();
+  // Assign user 1 the "Administrator" role, and make sure that generation of
```
Comment (klonos) [inline core/profiles/standard/standard.install]:
```suggestion
  // Assign the "Administrator" role to user 1, and make sure that generation of
```

**Review by ghost — APPROVED** [review]

**Comment by quicksketch** [comment]
Merged into 1.x and 1.20.x. Thanks @klonos and @BWPanda!

**Pull request merged**